ros2cli/ros2cli/daemon/__init__.py

```diff
@@ -56,6 +56,8 @@ def main(*, script_name='_ros2_daemon', argv=None):
             # expose getter functions of node
             server.register_function(
                 _print_invoked_function_name(node.get_node_names))
```

Comment:
Should this function be removed since it only provides incomplete information?

Reply:
I will remove it in the PR that updates `ros2lifecycle`